pkg/kubelet/kubelet.go

```diff
@@ -522,7 +507,7 @@ func NewMainKubelet(
 
 	klet.volumeManager, err = volumemanager.NewVolumeManager(
 		enableControllerAttachDetach,
-		hostname,
+		nodeName,
```

Comment:
I haven't seen any other side affect as the nodeName is only used in reconciler and to build cache of current state (volumes and pods that mount them)